schedule/yast/encryption/cryptlvm+cancel_existing_pvm.yaml

```diff
@@ -14,6 +14,7 @@ schedule:
   - installation/accept_license
   - installation/scc_registration
   - installation/encrypted_volume_activation
+  - console/validate_unencrypted_partition
```

Comment:
Hmm, I'm not sure about the name, because we validate that encrypted partition is not activated, so name is misleading. So I would propose something like `validate_encrypted_partition_not_activated`. WDYT?

Reply:
sounds good